src/CMakeLists.txt

```diff
@@ -116,26 +123,25 @@ add_library( ${TARGETLIB} SHARED
 # Include directories and compile options
 #
 
-set( LOCAL_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include )
-
-# Symlink ${YUI_INCLUDE_DIRS}
-# so the headers there can be included as <yui/YFoo.h>
-add_custom_target( local-include-dir
-  # check if the symlink already exists
-  COMMAND if [ ! -L "${LOCAL_INCLUDE_DIR}/yui" ]\; then
-      rm -rf ${LOCAL_INCLUDE_DIR}\;
-      mkdir  ${LOCAL_INCLUDE_DIR}\;
-      ln -s  ${YUI_INCLUDE_DIRS} ${LOCAL_INCLUDE_DIR}/yui\;
-    fi
-  )
-
-add_dependencies( ${TARGETLIB} local-include-dir )
-target_include_directories( ${TARGETLIB} BEFORE PUBLIC ${LOCAL_INCLUDE_DIR} )
+##- set( LOCAL_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include )
+##-
+##- # Symlink ${YUI_INCLUDE_DIRS}
+##- # so the headers there can be included as <yui/YFoo.h>
+##- add_custom_target( local-include-dir
+##-   # check if the symlink already exists
+##-   COMMAND if [ ! -L "${LOCAL_INCLUDE_DIR}/yui" ]\; then
+##-       rm -rf ${LOCAL_INCLUDE_DIR}\;
+##-       mkdir  ${LOCAL_INCLUDE_DIR}\;
+##-       ln -s  ${YUI_INCLUDE_DIRS} ${LOCAL_INCLUDE_DIR}/yui\;
+##-     fi
+##-   )
+##-
+##- add_dependencies( ${TARGETLIB} local-include-dir )
+##- target_include_directories( ${TARGETLIB} BEFORE PUBLIC ${LOCAL_INCLUDE_DIR} )
```

Comment:
This block is now dead code. Please remove.

Reply:
removed